src/Utils/generics.ts

```diff
@@ -386,4 +386,4 @@ export function trimUndefineds(obj: any) {
 	}
 
 	return obj
-}
```

Comment:
Should we keep the new line on end of file?

Reply:
yes